src/xdp/xsk.c

```diff
@@ -4058,6 +4082,19 @@ XskNotify(
         }
     }
 
+    if (InFlags & XSK_NOTIFY_FLAG_WAKE) {
+        //
+        // Wake any waiting thread.
+        //
+        if (!XskCancelIo(Xsk)) {
+            Status = STATUS_INVALID_DEVICE_STATE;
+            TraceError(
```

Comment:
I wouldn't trace this, at least not at error level: this path can easily be hit by unavoidable race conditions thousands of times/sec. We haven't been using WPP for data path tracing - manifested ETW is our practice there.